ngraph/cmake/external_npy.cmake

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Enable ExternalProject CMake module
+include(ExternalProject)
+
+#------------------------------------------------------------------------------
+# Download and install NPY ...
+#------------------------------------------------------------------------------
+
+SET(NPY_GIT_REPO_URL https://github.com/llohse/libnpy.git)
+
+# Build for ninja
+
+ExternalProject_Add(
+        ext_libnpy
+        PREFIX libnpy
+        GIT_REPOSITORY ${NPY_GIT_REPO_URL}
+        GIT_TAG "master"
```

Comment:
suggest adding TODO here. If this way accepted, we'll anyway need to checkout particular commit-id or release tag